wp-content/themes/amplify/amplify-feed.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Single Post Template: Amplify Feed
+ * Template Name: Amplify Feed Template
+ * Description: Shows the saved links feed that can be embedded on external sites.
+ */
+?>
+
+<!DOCTYPE html>
+<!--[if lt IE 7]> <html <?php language_attributes(); ?> class="no-js ie6"> <![endif]-->
+<!--[if IE 7]>    <html <?php language_attributes(); ?> class="no-js ie7"> <![endif]-->
+<!--[if IE 8]>    <html <?php language_attributes(); ?> class="no-js ie8"> <![endif]-->
+<!--[if IE 9]>    <html <?php language_attributes(); ?> class="no-js ie9"> <![endif]-->
+<!--[if (gt IE 9)|!(IE)]><!--> <html <?php language_attributes(); ?> class="no-js"> <!--<![endif]-->
+<head>
+	<meta charset="<?php bloginfo( 'charset' ); ?>" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <link rel="profile" href="https://gmpg.org/xfn/11" />
+    <link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>" />
+    <title>Amplify Embed Feed | INN Amplify Project</title>
+    <?php wp_head(); ?>
+</head>
+<body>
+
+    <div class="amplify-embed-feed">
+
+        <?php
+
+        if( isset( $_GET['tag'] ) ){
+            
+            $term = $_GET['tag'];
+            $term = get_term_by( 'slug', $term, 'lr-tags' );
+            
+            if ( function_exists ( 'largo_get_term_meta_post' ) ) {
+                
+                $term_meta_post = largo_get_term_meta_post( 'lr-tags', $term->term_id );
+                $term_meta = get_post_meta( $term_meta_post );
+
+                if( $term_meta['lr_more_link'] ){
+                    $term_more_link = $term_meta['lr_more_link'][0];
+                }
```

Comment:
This should be if ( ! empty( trim( $term_meta['lr_more_link'] ) ) ) { to account for an edge case where there's an empty string that is not false-y.

Reply:
Updated in https://github.com/INN/umbrella-inndev/pull/130/commits/e40d4b6141203c279a131b6abf63d4691e439a97.